draft-ietf-rats-architecture.md

```diff
@@ -1317,3 +1317,160 @@ use beyond the period for which it deems the Attestation Result to remain
+* Verify the Attestation Evidence and decided whether or not to allow application service
+
+This maps as follows in the RATS architecture:
+
+Endorser:
+
+: The operating system vendor is the endorser
+
+Endorsement:
+
+: The root certificate published by the endorser
+
+Attester:
+
+: The HW made by the device OEM and the SW made by the OS vendor
+
+Attestation Evidence:
+
+: A protocol message generated by the Attester on the request of the application that is then sent to the application vendor cloud service.
+
+Verifier and Verifier Owner:
+
+: are run by the application vendor cloud service
+
+Relying Party and Relying Party Owner:
+
+: are also the application vendor cloud service; since the Verify and Relying Part are the same the distinction between passport and background check doesn’t apply.
+
+Attestation Result:
+
+: non existent as the Verifier is the same as the Rely Party
+
+
+## System Example Made Up of Chip Vendor and IoT Device Maker
+
+Two entities (companies / organizations) participate in this example.
+
+The chip vendor that makes the chip. They also make the SW that is the Attester. It does the following:
+* Manufacturers the chip
+* Authors the SW that runs on the chip that is the Attester
+* Personalizes each chip with a unique 256-bit symmetric key and key ID
+* Also stores the 256-bit symmetric key in a well-secured database
+* Operates a web-based HMAC verification service the inputs to which are the key ID and HMAC to verify and the output is a yes or no on the verification
+
+The IoT device maker buys chips from the vendor, makes the chip the core of an IoT device (perhaps sensors for a home security service). It does the following:
+* Manufacturers and a device that an end user may purchase
+* Adds their own SW to the device over the top of the SW the chip vendor supplied
+* Operates a cloud-based service with which the device interacts
+* An Attestation Evidence is included whenever the IoT device communicates with the cloud service
+* The cloud service decodes the Attestation Evidence to get the key ID and computes the HMAC for it
+* The close service sends the key ID and HMAC to the chip vendors verification service
+
+This maps as follows in the RATS architecture:
+
+Endorser:
+
+: The chip vendor is the Endorser
+
+Endorsement:
+
+: The Endorsement only exists virtually inside the chip vendor
+
+Attester:
+
+: The HW and SW made by the chip vendor
+
+Attestation Evidence:
+
+: A protocol message generated by the Attester sent to the cloud service operated by the IoT device vendor
+
+Verifier and Verifier Owner:
+
+: The Verifier and Verifier Owner are split with part run by the chip vendor and part run by the IoT device maker
+
+Relying Party and Relying Party Owner:
+
+: are also the application vendor cloud service; since the Verify and Relying Part are the same the distinction between passport and background check doesn’t apply.
```

Comment:
```suggestion
: are also the application vendor cloud service; since the Verifier and Relying Party are part of the same service, this is the Background Check model.
```